paper/paper.md

```diff
@@ -61,54 +62,60 @@ bibliography: paper.bib
 In recent decades, improvements in algorithms, hardware, and theory have enabled crystalline materials to be studied computationally at the atomistic level with great accuracy and speed.
 To enable dissemination, reproducibility, and reuse, many digital crystal structure databases have been created and curated, ready for comparison with existing infrastructure storing structural characterizations of real crystals.
 These databases have been made available with bespoke application programming interfaces (APIs) to allow for automated and often open access to the underlying data.
-Such esoteric APIs incur maintenance and usability costs upon both the data providers and consumers, who may not be software specialists.
+Such esoteric and specialized APIs incur maintenance and usability costs upon both the data providers and consumers, who may not be software specialists.
 
-The OPTIMADE API specification [@andersen2021optimade; @OPTIMADE_spec], released in July 2020, aimed to reduce these costs by designing a common API for use across a consortium of collaborating materials databases.
+The OPTIMADE API specification [@andersen2021optimade; @OPTIMADE_spec], released in July 2020, aimed to reduce these costs by designing a common API for use across a consortium of collaborating materials databases and beyond.
 Whilst based on the robust JSON:API standard [@JSONAPI], the OPTIMADE API specification presents several domain-specific features and requirements that can be tricky to implement for non-specialist teams.
-The package presented here, `optimade-python-tools`, provides a modular reference server implementation and a set of associated tools to accelerate the development process for data providers, toolmakers and end-users.
+The repository presented here, `optimade-python-tools`, provides a modular reference server implementation and a set of associated tools to accelerate the development process for data providers, toolmakers and end-users.
 
 # Statement of need
 
-In order to accommodate existing materials database APIs, the OPTIMADE specification allows for flexibility in the specific data served but enforces a simple, but domain-specific, filter language on well-defined resources.
-This flexibility could be daunting to database implementers and maintainers and could act to increase the activation barrier to hosting an API.
+In order to accommodate existing materials database APIs, the OPTIMADE API specification allows for flexibility in the specific data served, but enforces a simple yet domain-specific filter language on well-defined resources.
+This flexibility could be daunting to database providers and maintainers, likely acting to increase the barrier and willingness for hosting an OPTIMADE REST API.
```

Comment:
I think I'd undo this bit, this sentence needs to reflect that the flexibility can be a blessing or a curse

Reply:
```suggestion
However, this flexibility could be daunting to database providers, likely acting to increase the barrier to hosting an OPTIMADE REST API.
```